docs/src/appendix/appendix.md

```diff
@@ -1,14 +1,11 @@
 ---
-layout: docs
-title:  "Appendix"
-section: "chisel3"
+sidebar_position: 3
 ---
 
 # Appendix
 
 This section covers some less-common Chisel topics.
 
-* [Differences between Chisel3 and Chisel2](chisel3-vs-chisel2)
```

Comment:
As far as I can tell, this document doesn't exist, it's a dead link on the current website.